tests/user/test_user_result_parser.py

```diff
@@ -289,13 +303,13 @@ def test_user_admin_can_parse_extract_user_base_omvs_csdata_success_xml(
         self,
         call_racf_mock: Mock,
     ):
-        user_admin = UserAdmin(
+        user_admin_local = UserAdmin(
```

Comment:
Again, I don't think it is necessary to call local variable local if they are already scoped. Leaving this as `user_admin` is fine.

Reply:
Fair enough, I wanted to make it more obvious this was a local version of it with different attributes, but that should be clear just by the fact that this is built within the test. Change made.